arch/src/aarch64/regs.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Arm Limited (or its affiliates). All rights reserved.
```

Comment:
I have a feeling that you are missing SPDX header here...

Reply:
Yes, you are right.